src/forward_kinematics.h

```diff
@@ -582,151 +582,167 @@ class RobotFK_Jacobian : public RobotFK_Fast
+                tip_dependencies[static_cast<size_t>(joint_model->getJointIndex()) * tip_count + tip_index] = 1;
             }
         }
     }
-    void computeJacobian(const std::vector<size_t>& variable_indices, Eigen::MatrixXd& jacobian)
-    {
-        double step_size = 0.00001;
-        double half_step_size = step_size * 0.5;
-        double inv_step_size = 1.0 / step_size;
-        auto tip_count = tip_frames.size();
-        jacobian.resize(tip_count * 6, variable_indices.size());
-        for(size_t icol = 0; icol < variable_indices.size(); icol++)
-        {
-            for(size_t itip = 0; itip < tip_count; itip++)
-            {
-                jacobian(itip * 6 + 0, icol) = 0;
-                jacobian(itip * 6 + 1, icol) = 0;
-                jacobian(itip * 6 + 2, icol) = 0;
-                jacobian(itip * 6 + 3, icol) = 0;
-                jacobian(itip * 6 + 4, icol) = 0;
-                jacobian(itip * 6 + 5, icol) = 0;
-            }
+    void computeJacobian(const std::vector<size_t>& variable_indices,
+                       Eigen::MatrixXd& jacobian) {
+    double step_size = 0.00001;
+    // double half_step_size = step_size * 0.5;
+    double inv_step_size = 1.0 / step_size;
+    auto tip_count = tip_frames.size();
+    jacobian.resize(static_cast<Eigen::Index>(tip_count * 6),
+                    static_cast<Eigen::Index>(variable_indices.size()));
+    for (Eigen::Index icol = 0;
+            icol < static_cast<Eigen::Index>(variable_indices.size()); ++icol) {
+        for (Eigen::Index itip = 0; itip < static_cast<Eigen::Index>(tip_count);
+            ++itip) {
+            jacobian(itip * 6 + 0, icol) = 0;
+            jacobian(itip * 6 + 1, icol) = 0;
+            jacobian(itip * 6 + 2, icol) = 0;
+            jacobian(itip * 6 + 3, icol) = 0;
+            jacobian(itip * 6 + 4, icol) = 0;
+            jacobian(itip * 6 + 5, icol) = 0;
         }
-        for(size_t icol = 0; icol < variable_indices.size(); icol++)
-        {
-            auto ivar = variable_indices[icol];
-            auto* var_joint_model = robot_model->getJointOfVariable(ivar);
-            if(var_joint_model->getMimic()) continue;
-            for(auto* joint_model : joint_dependencies[var_joint_model->getJointIndex()])
-            {
-                double scale = 1;
-                for(auto* m = joint_model; m->getMimic() && m->getMimic() != joint_model; m = m->getMimic())
-                {
-                    scale *= m->getMimicFactor();
-                }
-                auto* link_model = joint_model->getChildLinkModel();
-                switch(joint_model->getType())
-                {
-                case moveit::core::JointModel::FIXED:
-                {
-                    // fixed joint: zero gradient (do nothing)
+        }
+        for (size_t icol = 0; icol < variable_indices.size(); icol++) {
+        auto ivar = variable_indices[icol];
```

Comment:
Body of `for` loop is not indented.

Reply:
Turns out the `for` was intended one too many times. Addressed in [e9c01ee](https://github.com/PickNikRobotics/bio_ik/pull/1/commits/e9c01ee4ef00ced5d79bb788ab01a067e93e922c)